react/features/mobile/external-api/middleware.ts

```diff
@@ -186,18 +189,28 @@ externalAPIEnabled && MiddlewareRegistry.register(store => next => action => {
     }
 
     case ENDPOINT_MESSAGE_RECEIVED: {
-        const { participant, data } = action;
+        const { participant, data: json } = action;
+        const transcriptMessageID = json.message_id;
+        const { language: dataLanguage } = json;
 
-        if (data?.name === ENDPOINT_TEXT_MESSAGE_NAME) {
+        if (json?.name === ENDPOINT_TEXT_MESSAGE_NAME) {
             sendEvent(
                 store,
                 ENDPOINT_TEXT_MESSAGE_RECEIVED,
                 /* data */ {
-                    message: data.text,
+                    message: json.text,
                     senderId: participant.getId()
                 });
         }
 
+        notifyTranscriptionChunkReceived(
```

Comment:
Why are you doing this here as well, when it's already being called in the subtitles middleware?

Reply:
I thought that external api feature is sdk related and that is why I add it here as well.

Reply:
But you are already calling that function in the subtitles middleware, aren't you?

Reply:
Sure. That was understood from the start. I thought that for SDK we needed extra. Got confused a bit, but I am back on track.